ui/jobs/jobs.js

```diff
@@ -1563,6 +1564,73 @@ class Bars {
     };
   }
 
+  // (In Progress) (Need Help)
+  setupSch() {
+    let gcd = kUnknownGCD;
+
+    let BioBox = this.addProcBox({
```

Comment:
This is a tiny request, but would you mind naming your variables as `bioBox` (lowercase first, then camelcase) to match the rest of the code?